R/mpf_exporters.R

```diff
@@ -21,6 +21,8 @@
 #' @param paginate logical(1). Whether pagination should be performed,
 #'     defaults to \code{TRUE} if page size is specified (including
 #'     the default).
+#' @param ...  Further parameters to be passed to [paginate_to_mpfs()].
```

Comment:
Should we pass more things? all the pagination parameters are directly passed down and I think they are double documented

Reply:
No they are not double documented. The main docs are in `paginate_indices`. We might want to avoid the import of string parameters and be sure about the `...` use